controllers/user_controller.go

```diff
@@ -67,36 +67,37 @@ const (
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.14.1/pkg/reconcile
 func (r *S3UserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	logger := log.FromContext(ctx)
+	logger := log.FromContext(ctx).WithName("userCtrl")
```

Comment:
is there a way to not have to repeat ecah time the withName() ?

Reply:
Two-fold answer : 

- For every logger instantiated with `FromContext(ctx)`, I'm just dropping the `WithName()`, as it is actually redundant ; FromContext injects various values, among which `controller` and `controllerKind`, which are exactly what we need. Example : 

`2024-08-12T17:19:42.271+0200    INFO    The S3User CR s3u-test (or its owned Secret) has been removed. NOOP     {"controller": "s3user", "controllerGroup": "s3.onyxia.sh", "controllerKind": "S3User", "S3User": {"name":"s3u-test","namespace":"user-ozp6kz"}, "namespace": "user-ozp6kz", "name": "s3u-test", "reconcileID": "f8f1c63d-30fe-4691-a991-34221ba9337e"}`

- For the remaining two (one in path_controller.go and the other in interface.go for the s3client), we can't use `FromContext`, so I'm keeping something. But I am replacing `WithName` with `WithValues`, so the formatting of the logs is consistent with the one emanating from the controllers. 

Changes incoming in a minute or so.